nemo/collections/nlp/models/text_classification/text_classification_model.py

```diff
@@ -314,3 +316,54 @@ def list_available_models(cls) -> Optional[Dict[str, str]]:
     @classmethod
     def from_pretrained(cls, name: str):
         pass
+
+    def _prepare_for_export(self):
+        return self.bert_model._prepare_for_export()
+
+    def export(
+        self,
+        output: str,
+        input_example=None,
+        output_example=None,
+        verbose=False,
+        export_params=True,
+        do_constant_folding=True,
+        keep_initializers_as_inputs=False,
+        onnx_opset_version: int = 12,
+        try_script: bool = False,
+        set_eval: bool = True,
+        check_trace: bool = True,
+        use_dynamic_axes: bool = True,
+    ):
+        bert_model_onnx = self.bert_model.export(
+            'bert_' + output,
+            input_example,
```

Comment:
so the input example seems correct in this context.
But output is not, model's output is output of classifier

Reply:
Enforced recomputing both